components/admin/AdminFilter.tsx

```diff
@@ -0,0 +1,155 @@
+      <div className="flex flex-wrap gap-1 py-3 pr-9">
+        {filters &&
+          filters.map((filter, i) => {
+            const focusNotApproved = filter.status !== StatusEnum.APPROVED;
+            return (
+              <div className={filterClass}>
+                <span
+                  className={cn(focusNotApproved && `bg-violet-600/20 font-medium text-violet-600`)}
+                  key={memberId + filter.id}
+                >
+                  {filter.name}
+                  {focusNotApproved ? ` (${filter.status})` : null}
+                </span>
+              </div>
+            );
+          })}
+        {suggestedFilter && !unnapprovedFilters.includes(suggestedFilter) && (
+          <div className={filterClass}>
+            <span className={cn(`bg-violet-600/20 font-medium text-violet-600`)} key={suggestedFilter}>
+              {suggestedFilter} (pending approval)
+            </span>
+          </div>
+        )}
+        <Button className="absolute right-10 p-2" variant="secondary" size="sm" onClick={handleOpen}>
+          ✏️
+        </Button>
+        <Dialog open={open} onOpenChange={setOpen}>
+          <DialogContent className="gap-0 p-0 outline-none bg-white">
+            <DialogHeader className="px-4 pb-4 pt-5">
+              <DialogTitle className="text-2xl">
+                {filterTypePlural} <span className="text-3xl">{emoji}</span>
+              </DialogTitle>
+              <DialogDescription>
+                Edit your {filterTypePlural.toLowerCase()} list, you can have up to 3
+              </DialogDescription>
+            </DialogHeader>
+            <Command className="h-80 border-t">
+              <CommandInput placeholder={"Search " + filterTypePlural.toLowerCase()} className="h-9" />
+              <CommandEmpty>No {filterTypePlural.toLowerCase()} found.</CommandEmpty>
+              <CommandGroup className="overflow-auto">
+                {allFilters.map((filter) => (
+                  <CommandItem key={filter.id} value={filter.name} onSelect={() => handleSelect(filter)}>
+                    {filter.name}
+                    <CheckIcon
+                      className={cn(
+                        "ml-auto h-4 w-4",
+                        selectedFilters.includes(filter.id) ? "opacity-100" : "opacity-0",
+                      )}
+                    />
+                  </CommandItem>
+                ))}
+              </CommandGroup>
+            </Command>
+            <DialogFooter className="border-t p-4 sm:justify-start">
+              <DialogHeader>
+                {!suggestOpen ? (
+                  <>
+                    <Button variant="secondary" size="sm" onClick={() => setSuggestOpen(true)}>
+                      Suggest a new {filterTypeSingular}
+                    </Button>
+                  </>
+                ) : (
+                  <>
+                    <DialogTitle className="text-sm">Please suggest with care 🤙🏽</DialogTitle>
```

Comment:
Is there an easy way for us to allow removing a suggestion? I'm trying this and not sure if there's a way.

Reply:
Technically there isn't right now. Industry and focus reviewing (in my head) is a different feature set separate from member reviews, but I _can_ include that in this PR. What do you think?

Reply:
Nah I think we can save for when that's necessary. See how many suggestions come in through the edit field and adapt based on that.

(AFK so I cannot check) but for us knowing's sake, can they overwrite a suggestion if they send something up they wish to hide? Like accidentally send my password or something weird as a suggestion, let them obfuscate it. If not also probably no worries but something to keep in mind for a follow up PR.

Reply:
Sure, I think you're talking about it from the **member** side of things, like maybe a member wants to retract a suggestion. I'm talking about it from the **admin** side of things which I think are both valid use cases for removing a suggestion.

I know we still have to handle the whole **hawaiiansintech** github group repo thing, but I created it as an issue here: https://github.com/hawaiiansintech/hawaiiansintech/issues/31